nautobot/utilities/forms/fields.py

```diff
@@ -458,3 +458,69 @@ def prepare_value(self, value):
         if value is None:
             return ""
         return json.dumps(value, sort_keys=True, indent=4)
+
+
+class JSONArrayFormField(forms.JSONField):
```

Comment:
This feels like a lot of boilerplate wrapped around the validation of the underlying field type. Can you please provide a brief docstring to explain why all of this is needed? :)

Reply:
It would be great to have docstrings on each of the methods below as well so that the reader doesn't have to keep referring back to the Django documentation to understand what each of them is for. :-)